package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dataesr/react-dsfr",
-  "version": "3.4.3",
+  "version": "3.5.0",
```

Comment:
Is there any reason why you change the lib version manually ? It will be automatically done if you use the correct commit message, please have a look at the readme https://github.com/dataesr/react-dsfr#release-policy-and-package-publishing. Let me know if it is not clear.

Reply:
You're right, I was too focused on the breaking change stuff and doing it properly, I went over the semantic-release. I'll fix it

Reply:
Voila ! Thank you for your review